lib/proxies/iot.rest.js

```diff
@@ -186,6 +186,31 @@ IoTKitRestCloud.prototype.getCatalog = function (data, callback) {
     });
 };
 
+IoTKitRestCloud.prototype.getDevice = function (data, callback) {
+    var me = this;
+    me.logger.debug("Getting device : ");
+    var token = data.deviceToken;
+    delete data.deviceToken;
+    var did = data.deviceId;
+    delete data.deviceId;
+    var dataPayload = {
+        deviceId : did,
+        deviceToken: token
```

Comment:
why you dont use data.deviceToken & data.deviceId instead of new variables? Even if you need to delete parts of object, you can do it later